l3kernel/l3file.dtx

```diff
@@ -917,6 +917,21 @@
 %   function. An error is raised if the file is not found.
 % \end{function}
 %
+% \begin{function}[added = 2023-05-18, EXP]{\file_input_raw:n}
+%   \begin{syntax}
+%     \cs{file_input_raw:n} \Arg{file name}
+%   \end{syntax}
+%   Searches for \meta{file name} in the path as detailed for
+%   \cs{file_if_exist:nTF}, and if found reads in the file as
+%   additional \TeX{} source. No data concerning the file is
+%   tracked. If the file is not found, no action is taken.
+%   \begin{texnote}
```

Comment:
> no action is taken.

sounds like it is skipped (like `\@input`) but I think you mean "no action is taken _by latex_" so  you get an engine level file not found error.

Reply:
Er, not: we have at this point applied `\file_full_name:n` and got an empty result, so we branch to using `\tl_if_blank:nF` and so a missing file will be entirely skipped.

Reply:
oops, sorry I should have noticed `full_name_` did that.

This seems wrong, normalising the name  is  needed for assigning the correct hook if you have hooks, but here to be honest  I think I'd just let it to the primitive, or at most define it as `\def\file_input_raw:n#1{\@@input#1\relax}` so braces work on old releases.

The main use case here is to underlie a document level input command that can be used in tables, but if this one skips missing file errors, I think any document level command would have to not use this, and be defined directly from the primitive.

Reply:
I've adjusted to issue an expandable error - so something we still control